common/src/test/java/org/conscrypt/MlDsaTest.java

```diff
@@ -148,87 +159,131 @@ public void mldsa87_works() throws Exception {
         assertEquals("ML-DSA", privateKey.getAlgorithm());
         assertEquals("ML-DSA", publicKey.getAlgorithm());
 
-        byte[] msg = new byte[123];
-        Signature ss = Signature.getInstance("ML-DSA-87", conscryptProvider);
-        ss.initSign(privateKey);
-        ss.update(msg);
-        byte[] sig = ss.sign();
-        assertEquals(4627, sig.length);
+        for (String signatureAlgorithm : new String[] {"ML-DSA-87", "ML-DSA"}) {
+            byte[] msg = new byte[123];
+            Signature ss = Signature.getInstance(signatureAlgorithm, conscryptProvider);
```

Comment:
What happens here now if signatureAlgorithm == "ML-DSA"? Will this sign with ML-DSA 87 or 65?

Should we test that sign with ML-DSA-87 also verifies with ML-DSA?